pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = [
+    "cmake>=3.19",
+    "numpy>=1.21.2",
+    "scikit-build-core >=0.4.3",
+    "ninja; platform_system!='Windows'"
+]
+
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "finufft"
+version = "0.0.1"
+description = "Python bindings for the FINUFFT C++ library"
+readme = "README.md"
+requires-python = ">=3.8"
+
+[tool.scikit-build]
+# Protect the configuration against future changes in scikit-build-core
+minimum-version = "0.4"
+cmake.targets = ["finufft"]
+cmake.args = ['-DFINUFFT_BUILD_PYTHON=ON', '-DCMAKE_BUILD_TYPE=Release'] # not sure if this is necessary
+wheel.packages = ["python/finufft/finufft"]
+wheel.exclude = ["**.pyx"]
+# Setuptools-style build caching in a local directory
+build-dir = "build/{wheel_tag}"
+# Build stable ABI wheels for CPython 3.8+
+wheel.py-api = "cp38"
```

Comment:
Why do we need this? Shouldn't we just leave it to the user (or the CI pipeline) to specify which ABI they want?

Reply:
not sure it is necessary. Maybe for cross platform builds? The wheel is built using python 3.10 but we want it to work on previous versions too